src/gui/tray/UserLine.qml

```diff
@@ -163,6 +182,15 @@ MenuItem {
                                 color: parent.parent.hovered ? Style.lightHover : "transparent"
                             }
                         }
+
+                        Accessible.role: Accessible.Button
+                        Accessible.name: isConnected ? qsTr("Log out") : qsTr("Log in")

```

Comment:
Good Accessible.name, doesn’t need a long description either I would say. 

Please correct me if I’m wrong @tyrylu @Adriorjalesvidal: I would say if the Accessible.name is "Log out" or "Log in", we don’t need a description, right?